platform/src/auth/mod.rs

```diff
@@ -1,8 +1,31 @@
 use async_trait::async_trait;
 use serde::{Deserialize, Serialize};
+use aws_config::environment::EnvironmentVariableRegionProvider;
+use aws_config::meta::region::RegionProviderChain;
+use aws_config as sdk;
+use aws_sdk_secretsmanager::{Client, Error as awsError};
 
 use crate::Error;
 
+pub async fn get_secret(secret_name: &str) -> Result<Option<String>, awsError> {
```

Comment:
After talking to Richard I think we'll delete this yeah?

Reply:
My understanding is we still need to iron out the alternative first, but yeah it needs to be replaced with the approved way